src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -4040,6 +4040,12 @@ margin_right = 345.0
 margin_bottom = 203.0
 mouse_filter = 2
 
+[node name="to be implemented" type="Label" parent="GroupHolder/organelleSelection/thermoplast/MarginContainer/VBoxContainer"]
```

Comment:
Also the label should use Lato font similar to the `Description` label.

Reply:
I suggested copy-pasting the existing "TO_BE_IMPLEMENTED" node, here, which seems to correctly have the custom font set: https://github.com/Revolutionary-Games/Thrive/blob/f8ecf2065c2ed5f09fb9959fa10cd4782b75ce2c/src/gui_common/tooltip/ToolTipManager.tscn#L2962